sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/DigitalTwinsRelationshipTest.java

```diff
@@ -0,0 +1,231 @@
+            String floorTwinContainedInRelationshipPayload = getRelationshipPayload(floorTwinId, CONTAINED_IN_RELATIONSHIP);
+            String floorCooledByHvacPayload = getRelationshipPayload(hvacTwinId, COOLED_BY_RELATIONSHIP);
+            List<Object> floorContainsRoomUpdatePayload = getRelationshipUpdatePayload("/isAccessRestricted", false);
+
+            // Create relationship from Floor -> Room
+            BasicRelationship floorRoomRelationship = client.createRelationship(floorTwinId, FLOOR_CONTAINS_ROOM_RELATIONSHIP_ID, floorContainsRoomPayload, BasicRelationship.class);
+            assertThat(floorRoomRelationship.getId())
+                .isEqualTo(FLOOR_CONTAINS_ROOM_RELATIONSHIP_ID)
+                .as("Created relationship from floor -> room");
+            logger.info("Created {} relationship between source = {} and target = {}", floorRoomRelationship.getId(), floorRoomRelationship.getSourceId(), floorRoomRelationship.getTargetId());
+
+            // Create relationship from Floor -> Hvac
+            BasicRelationship floorHvacRelationship = client.createRelationship(floorTwinId, FLOOR_COOLED_BY_HVAC_RELATIONSHIP_ID, floorCooledByHvacPayload, BasicRelationship.class);
+            assertThat(floorHvacRelationship.getId())
+                .isEqualTo(FLOOR_COOLED_BY_HVAC_RELATIONSHIP_ID)
+                .as("Created relationship from floor -> hvac");
+            logger.info("Created {} relationship between source = {} and target = {}", floorHvacRelationship.getId(), floorHvacRelationship.getSourceId(), floorHvacRelationship.getTargetId());
+
+            // Create relationship from Hvac -> Floor
+            BasicRelationship hvacFloorRelationship = client.createRelationship(hvacTwinId, HVAC_COOLS_FLOOR_RELATIONSHIP_ID, floorTwinCoolsRelationshipPayload, BasicRelationship.class);
+            assertThat(hvacFloorRelationship.getId())
+                .isEqualTo(HVAC_COOLS_FLOOR_RELATIONSHIP_ID)
+                .as("Created relationship from hvac -> floor");
+            logger.info("Created {} relationship between source = {} and target = {}", hvacFloorRelationship.getId(), hvacFloorRelationship.getSourceId(), hvacFloorRelationship.getTargetId());
+
+            // Create relationship from Room -> Floor
+            BasicRelationship roomFloorRelationship = client.createRelationship(roomTwinId, ROOM_CONTAINED_IN_FLOOR_RELATIONSHIP_ID, floorTwinContainedInRelationshipPayload, BasicRelationship.class);
+            assertThat(roomFloorRelationship.getId())
+                .isEqualTo(ROOM_CONTAINED_IN_FLOOR_RELATIONSHIP_ID)
+                .as("Created relationship from room -> floor");
+            logger.info("Created {} relationship between source = {} and target = {}", roomFloorRelationship.getId(), roomFloorRelationship.getSourceId(), roomFloorRelationship.getTargetId());
+
+            // Create a relation which already exists - should return status code 409 (Conflict).
```

Comment:
Could this be split out into a separate test?

Reply:
I thought about that, but that would require that we create the model and twins again. Since they were already available as a part of this lifecycle test, I added this assertion in here directly.